indico/modules/rb/client/js/components/BookingBootstrapForm.jsx

```diff
@@ -291,9 +291,11 @@ class BookingBootstrapForm extends React.Component {
               isOutsideRange={dt => {
                 return !isBookingStartDateValid(dt, isAdminOverrideEnabled, bookingGracePeriod);
               }}
+              withFullScreenPortal={window.innerHeight < 730}
```

Comment:
We have a `responsiveReactDates` util where you can use the actual react-responsive utils to do this..

Reply:
But wouldn't that affect all date components that depend on this? 
I could, however, re-use a tablet dimension, even though we're using height here instead.

Reply:
True.. depends on whether we want it for all responsive date components. If not, maybe add a prop to override the dimension if that can be done easily?